src/plugins/controls/public/control_group/embeddable/control_group_container.tsx

```diff
@@ -104,25 +105,35 @@ export class ControlGroupContainer extends Container<
     buttonType: CreateControlButtonTypes,
     closePopover?: () => void
   ) => {
+    const ControlsServicesProvider = pluginServices.getContextProvider();
+
     return (
-      <CreateControlButton
-        buttonType={buttonType}
-        defaultControlWidth={this.getInput().defaultControlWidth}
-        defaultControlGrow={this.getInput().defaultControlGrow}
-        updateDefaultWidth={(defaultControlWidth) => this.updateInput({ defaultControlWidth })}
-        updateDefaultGrow={(defaultControlGrow: boolean) =>
-          this.updateInput({ defaultControlGrow })
-        }
-        addNewEmbeddable={(type, input) => this.addNewEmbeddable(type, input)}
-        closePopover={closePopover}
-        getRelevantDataViewId={() => this.getMostRelevantDataViewId()}
-        setLastUsedDataViewId={(newId) => this.setLastUsedDataViewId(newId)}
-      />
+      <ControlsServicesProvider>
+        <CreateControlButton
+          buttonType={buttonType}
+          defaultControlWidth={this.getInput().defaultControlWidth}
+          defaultControlGrow={this.getInput().defaultControlGrow}
+          updateDefaultWidth={(defaultControlWidth) => this.updateInput({ defaultControlWidth })}
+          updateDefaultGrow={(defaultControlGrow: boolean) =>
+            this.updateInput({ defaultControlGrow })
+          }
+          addNewEmbeddable={(type, input) => this.addNewEmbeddable(type, input)}
+          closePopover={closePopover}
+          getRelevantDataViewId={() => this.getMostRelevantDataViewId()}
+          setLastUsedDataViewId={(newId) => this.setLastUsedDataViewId(newId)}
+        />
+      </ControlsServicesProvider>
```

Comment:
Because we are no longer using `getServices` in `create_control.tsx`, we need to wrap the entire component in the services provider so that, when we call ` pluginServices.getHooks()` at the start of that file, the provider is set up **regardless** of whether the callout button is clicked or the button in the toolbar.